File: python/examples/01_health.py
"""
Example 01

Health Check
"""

from __future__ import annotations

import json

from parmana import ParmanaClient


def main() -> None:
    client = ParmanaClient(
        endpoint="http://localhost:3000",
    )

    health = client.execution.health()

    print(
        json.dumps(
            health,
            indent=2,
        )
    )


if __name__ == "__main__":
    main()

File: typescript/examples/01-health.ts

import {
  HttpTransport,
  ParmanaClient,
} from "@parmana/typescript-sdk";

const client =
  new ParmanaClient({
    endpoint:
     "http://localhost:3000",

    transport:
      new HttpTransport({
        endpoint:
         "http://localhost:3000",
      }),
  });

const health =
  await client.health();

console.log(health);
